src/core/collections.js

```diff
@@ -74,12 +74,18 @@ export const initCollection = screen => key => {
 		});
 	};
 
+	const getUnique = ({ key, value }) => catalogue.find(item => item[key] === value);
+
+	const getSelected = () => getUnique({ key: "selected", value: true });
```

Comment:
This to be more generic.

Collections are (supposed to be) agnostic so having a selection specific method for a tag starts adding hard dependencies between the select screen's domain and the collection domain (I am aware the `qty` value breaks this idea though :) )